mistune/scanner.py

```diff
@@ -128,7 +128,10 @@ def iter(self, string, state, parse_text):
             yield parse_text(string[last_end:], state)
 
 
-def escape(s, quote=True):
+def escape(s : str, quote : str = True) -> str:
```

Comment:
please don't add type hint. Mistune is compatible with Python 2.7

Reply:
Good, but then the docstring has to be different. 
Fixed in aa22836.